src/shared/utils/bundleAnalysis.ts

```diff
@@ -57,3 +57,21 @@ export const formatTimeToString = (milliseconds: number) => {
     unit: 'millisecond',
   }).format(milliseconds)
 }
+
+export const localizeBundleSize = (bytes: number) => {
+  const positiveBytes = Math.abs(bytes)
+
+  if (positiveBytes < KILOBYTE) {
+    return bytes
+  }
+
+  if (positiveBytes >= KILOBYTE && positiveBytes < MEGABYTE) {
+    return bytes / KILOBYTE
```

Comment:
We're okay with decimal numbers for this fn yeah?

Reply:
I believe this should always return an integer unless the input is not an integer. And I think the input will always be an integer

Reply:
Unless there's some weird JS stuff going on lol

Reply:
We only ever pass in integer values here, however we could get decimals returned which is fine, as we use rounding, ceil, floor, and Intl.NumberFormat to handle the values and remove the extra decimals

Reply:
cool cool, yeah in like 3/2 you'd get 1.5, so wanted to confirm we wanted this, which makes sense anyway

Reply:
if you look at the top of that file we have some consts defined so we're only dividing by multiples of 10 only moving the decimal place